spec/haskell/src/SEL4/Object/CNode.lhs

```diff
@@ -338,7 +338,7 @@ Determine whether the CTE is valid, and has children to be deleted. If a child i
 >         nextCTE <- withoutPreemption $ getCTE nextPtr
 >         when (cte `isMDBParentOf` nextCTE) $ do
 >             cteDelete nextPtr True
->
+>             preemptionPoint
```

Comment:
Is this supposed to be part of the current PR? It isn't mentioned in the description and doesn't seem related to reply queues.

Reply:
This is the commit that Gerwin merged this morning. I included it here so that the Refine test won't fail because of it missing. It will go away when I rebase later. (I kind of hoped that github will do that automatically..)